quetz/deps.py

```diff
@@ -92,17 +92,22 @@ def get_tasks_worker(
     else:
         worker = "thread"
 
+    if config.configured_section("redis"):
+        redis_ip = config.redis_ip
+        redis_port = config.redis_port
+        redis_db = config.redis_db
+    else:
+        redis_ip = "127.0.0.1"
```

Comment:
I am not sure whether these are reasonable defaults, perhaps we should force the user to provide the values if the worker is set to redis?

Reply:
Yeah, we can do that too. The default `IP`, `Port`, and `db` are always those if you run a server `locally`.

Reply:
so maybe we can keep them, but the defaults should be handled in quetz.config.Config, not here